packages/angular_devkit/build_optimizer/src/transforms/scrub-file.ts

```diff
@@ -8,6 +8,9 @@
 import * as ts from 'typescript';
 import { collectDeepNodes } from '../helpers/ast-utils';
 
+const tslibDecorateRe = /\btslib(?:_\d+)?\.__decorate\b/;
+const tslibRe = /\btslib(?:_\d+)?\b/;
```

Comment:
This name can vary, so it would be safer to look up the import rathar than use a regex. The same is done for Angular symbols in `identifierIsMetadata`+`findAngularMetadata`.